runner/internal/reflectutil/reflectutil.go

```diff
@@ -71,3 +69,24 @@ func safeMatchFunc(matchFunc MatchFunc, pathname string) func(string) bool {
 		return key == pathname
 	}
 }
+
+func propertyByNameFunc(host reflect.Value, match func(string) bool) reflect.Value {
+	if fieldMatch := host.FieldByNameFunc(match); fieldMatch.IsValid() {
+		return fieldMatch
+	}
+	if methodMatch := methodByNameFunc(host, match); methodMatch.IsValid() {
+		return methodMatch.Call([]reflect.Value{})[0]
```

Comment:
This could produce unexpected behavior if called on a method which take 1 or more argument.
It does not happen at the moment because it is only called around an `Aggregate` and a `Field` defining a method respecting the limitation.

Should we prevent unexpected behavior? And can we even do that easily?

Reply:
Yup that's entirely true. We could use some error handling here, but it's an internal after all and I was to lazy to bother with that 😛

Reply:
Then let's deal with that later 👍